<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpawnParameterMissileLauncherWidget</class>
 <widget class="QWidget" name="SpawnParameterMissileLauncherWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Orientation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="orientationComboBox">
       <item>
        <property name="text">
         <string>0: Top-right to bottom-left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1: Vertical</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2: Top-left to bottom-right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3: Top-right to bottom-left</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Metatile coverage</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="metatileCoverageSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Facing</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="facingLeftButton">
         <property name="text">
          <string>Left</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="facingRightButton">
         <property name="text">
          <string>Right</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
